go/vt/schemadiff/errors.go

```diff
@@ -423,3 +423,33 @@ type EntityNotFoundError struct {
 func (e *EntityNotFoundError) Error() string {
 	return fmt.Sprintf("entity %s not found", sqlescape.EscapeID(e.Name))
 }
+
+type EnumValueOrdinalChangedError struct {
+	Table      string
+	Column     string
+	Value      string
+	Ordinal    int
+	NewOrdinal int
+}
+
+func (e *EnumValueOrdinalChangedError) Error() string {
+	return fmt.Sprintf("ordinal of %s changed in enum or set column %s.%s, from %d to %d", e.Value, sqlescape.EscapeID(e.Table), sqlescape.EscapeID(e.Column), e.Ordinal, e.NewOrdinal)
+}
+
+type UnknownColumnCharsetCollationError struct {
+	Column  string
+	Charset string
+}
+
+func (e *UnknownColumnCharsetCollationError) Error() string {
+	return fmt.Sprintf("unable to determine collation for column %s with charset %s", sqlescape.EscapeID(e.Column), e.Charset)
```

Comment:
Might be worth using %q instead of %s here and below, in case they do something crazy like use a column name with whitespace in it (which you can do with backticks).

Reply:
Columns escaping is already taken care of by `sqlescape.EscapeID(e.Column)`, which adds backticks. Added `sqlescape.EscapeID(e.Column)` test case with spaces. Changed to `%q` for charset/collation.